### ISSUE TITLE
Undeclared identifier compile error in VS2017

### ISSUE THREAD

**Issue by mohkamfer** [symless/synergy-core#6376]
File src/lib/arch/win32/ArchSystemWindows.cpp at line 144 It contains _ULonglong which I don't really know in which library/sdk it lives, but it's undefined and none of the header files contain its definition.

If replaced with 'long' the solution builds without errors.
Not really sure if that's a bug, so I'm not adding any tags.

EDIT: I don't really remember checking out a branch, so this should be on master.

**Comment by nbolton** [comment]
As far as I can tell, the `getLibsUsed` isn't used. We could just delete that function.

**Comment by nbolton** [comment]
@mohkamfer Hey, saw this when trying to compile a VS2017 project. Are you also seeing issue #6379?

**Comment by mohkamfer** [comment]
@nbolton Nope didn't encounter that one, just as I fixed the _ULonglong issue all the projects compiled successfully. Maybe this can be related to Windows SDK's version?

On a side note I lost interest in the software as my whole setup is wireless and control stutters and hiccups like there's no tomorrow :angry:  so frustrated.

**Comment by nbolton** [comment]
Does the issue go away when using Ethernet? If so, it could be an issue with your WiFi adapter.

**Comment by hmoenck** [comment]
I encountered this error as well. I was able to monkey-patch it by replacing "_ULongLong" with "unsigned long long" - it builds just fine. For the context of usage it seems sane: PID's should not be larger than even a regular int. Also this is used for generating message string in whatever context (unlikely to be breaking even in the unlikely event of being faulty).  
Didn't yet test this fix in production though. 
Built on Win7 x64 with VS2017, cl 19.15.26729
